cmd/util/cmd/system-addresses/cmd.go

```diff
@@ -0,0 +1,62 @@
+package addresses
+
+import (
+	"bytes"
+	"sort"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/fvm/systemcontracts"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagChain     string
+	flagSeparator string
+)
+
+var Cmd = &cobra.Command{
+	Use:   "system-addresses",
+	Short: "print addresses of system contracts",
+	Run:   run,
+}
+
+func init() {
+	Cmd.Flags().StringVar(&flagChain, "chain", "", "Chain name")
+	_ = Cmd.MarkFlagRequired("chain")
+
+	Cmd.Flags().StringVar(&flagSeparator, "separator", ",", "Separator to use between addresses")
```

Comment:
Is this format for your specific use case?  I think it would also be useful to print the contract name along with its addresses.

Reply:
It's just a comma-separated list of addresses. For now names are not needed. If they are needed, they can be added later